ios/Podfile.lock

```diff
@@ -1055,4 +1055,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 48093a300788c218e76f70cf6d102915cd993f34
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.0
```

Comment:
@s77rt Why is this change required?

Reply:
Should we revert it? On my end, on      'pod install', it is reverted to 1.11.3

Reply:
It depends on the version you are using. I think we will all update to 1.12.0 at some point so no harm in leaving this as it does not break anything (been using this version for a while) and the Podfile checklist still passes.
Version 1.12.0 was released 3 weeks ago https://github.com/CocoaPods/CocoaPods/releases.

Reply:
@s77rt I will suggest not doing that from this PR. I believe we should update it from different PR for traceability.

Reply:
Sure, please suggest the change based on your podfile

Reply:
```suggestion
COCOAPODS: 1.11.3
```

Reply:
Thanks!